tests/test_tls.py

```diff
@@ -0,0 +1,32 @@
+import pytest
+import datajoint as dj
+from pymysql.err import OperationalError
+
+
+def test_secure_connection(db_creds_test, connection_test):
```

Comment:
Have to use `connection_test` fixture so that the test user is created in the DB.